example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -321,9 +304,23 @@ describe('When there are granule differences and granule reconciliation is run',
       .toBe(1);
   });
 
+  it('deletes a reconciliation report through the Cumulus API', async () => {
+    await reconciliationReportsApi.deleteReconciliationReport({
+      prefix: config.stackName,
+      name: reportRecord.name
+    });
+
+    reconciliationReportsApi.getReconciliationReport({
```

Comment:
Should this be testing that the report doesn't exist?  Maybe it is.  Can you show that more explicitly somehow?

Reply:
Thanks for catching this.  I updated the test.